ow_faults_injection/include/ow_faults_injection/JointsFaults.h

```diff
@@ -15,13 +15,13 @@
 
 struct JointFaultInfo
 {
-  const std::string jointLockedFault;
+  const std::string fault;
   bool activated;
   double friction;
   unsigned int landerJoint;
 
   JointFaultInfo(const std::string& fault_, unsigned int lander_joint, bool activated_ = false, double friction_ = 0.0)
-  : jointLockedFault(fault_ + "_joint_locked_failure"), 
+  : fault(fault_ + "_failure"),
```

Comment:
This would break things.

Faults currently has the following format:
```
dist_pitch_joint_locked_failure
hand_yaw_joint_locked_failure
prox_pitch_joint_locked_failure
scoop_yaw_joint_locked_failure
shou_pitch_joint_locked_failure
shou_yaw_joint_locked_failure
```

Lanssie made some changes when constructing the `m_JointsFaultsMap` map such that she only need to specify the joint name without the *_joint_locked_failure* postfix

Reply:
fixed